components/team/Members.tsx

```
@@ -1,8 +1,14 @@
-import { Error, LetterAvatar, Loading } from '@/components/shared';
+import {
+  EmptyState,
+  Error,
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)